lib/rules/no-jquery-methods.js

```diff
@@ -24,6 +24,14 @@ const BLACKLIST = [
   'wrapAll', 'wrapInner'
 ];
 
+function isJQueryCallee(node) {
```

Comment:
@trentmwillis this will cover us accidentally blacklisting non-jQuery object methods. Ensuring we allow for at a min your example:

```
const myJQueryObj = this.$();
myJQueryObj.add(foo);
```